articles/alternate-apple-mdm-url.md

```diff
@@ -0,0 +1,55 @@
+# Configuring an Alternative Apple MDM URL
+
+![Fleet MDM Configuration](../website/assets/images/articles/configuring-alternative-mdm-url-800x450@2x.png)
```

Comment:
placeholder PNG.  could use some help finding a good one @rachaelshaw

Reply:
^^ @mike-j-thomas

Reply:
@rachaelshaw, @mostlikelee, no need to include a graphic for guides and articles. It takes too much time for the author and myself and is hard to maintain consistency with them. I'll double-check the handbook to see if that's documented—it could have just been a verbal understanding between JD and me.